### PR TITLE
Add 2 new command to Reco Pack

### PR DESCRIPTION
**Pull request by GalNakash-RecoLabs**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by mchasepan** [comment]
@GalNakash-RecoLabs may I suggest you also fix on your [readme](https://github.com/demisto/content/blob/master/Packs/Reco/README.md) 
- And add the new command description 
- Fix the typo "Abnormal Security"

**Comment by GalNakash-RecoLabs** [comment]
> @GalNakash-RecoLabs may I suggest you also fix on your [readme](https://github.com/demisto/content/blob/master/Packs/Reco/README.md)
> 
> * And add the new command description
> * Fix the typo "Abnormal Security"

Fixed thanks

File: Packs/Reco/Integrations/Reco/Reco_test.py [hunk 1]
```diff
@@ -212,3 +265,54 @@ def test_max_fetch():
     max_fetch = 200
     result = get_max_fetch(max_fetch)
     assert result == max_fetch
+
+
+def test_update_reco_incident_timeline(requests_mock, reco_client: RecoClient) -> None:
+    incident_id = uuid.uuid1()
+    requests_mock.put(f"{DUMMY_RECO_API_DNS_NAME}/incident-timeline/{str(incident_id)}", json={}, status_code=200)
+    reco_client.update_reco_incident_timeline(incident_id=incident_id, comment="test")
+
```
Comment (yucohen) [inline Packs/Reco/Integrations/Reco/Reco_test.py:274]: Maybe we can add an assertion here in order to test that the command actually inserts the right data into the outputs - relevant to all unit tests added

**Review by yucohen — CHANGES_REQUESTED** [review]
Hi @GalNakash-RecoLabs Thanks for your contribution! Great work.
Please see my comments.
Also note that the validation step fails on:
`[ERROR]: Packs/Reco/Integrations/Reco/Reco.yml: [IN151] - The arguments of the integration command `reco-get-risky-users` can not be None. If the command has no arguments, use `arguments: []` or remove the `arguments` field.
To fix the problem, try running `demisto-sdk format -i Packs/Reco/Integrations/Reco/Reco.yml `

[WARNING]: Packs/Reco/Integrations/Reco/Reco.yml: [DS104] - No detailed description file was found. Consider adding one.

[ERROR]: Packs/Reco/Integrations/Reco/Reco.yml: [RM110] - The following commands appear in Reco.yml but not in the README file:
update-reco-incident-timeline
resolve-visibility-event
reco-get-risky-users
reco-add-risky-user`
try to fix those before the demo :)
Feel free to reach out here or on slack if you have any questions

**Comment by GalNakash-RecoLabs** [comment]
> update-reco-incident-timeline resolve-visibility-event reco-get-risky-users reco-add-risky-user

Thanks, I fixed most of the issues. When execute `demisto-sdk format -i Packs/Reco/Integrations/Reco/Reco.yml
` I got the following error:
`ook_validations/pack_unique_files.py", line 1010, in are_valid_files
    self.validate_pack_meta_file()
  File "/Users/galnakash/.local/share/virtualenvs/content-NObhtT8p/lib/python3.10/site-packages/demisto_sdk/commands/common/hook_validations/pack_unique_files.py", line 429, in validate_pack_meta_file
    self._is_right_version(),
  File "/Users/galnakash/.local/share/virtualenvs/content-NObhtT8p/lib/python3.10/site-packages/demisto_sdk/commands/common/hook_validations/base_validator.py", line 46, in wrapper
    return func(self, *args, **kwargs)
  File "/Users/galnakash/.local/share/virtualenvs/content-NObhtT8p/lib/python3.10/site-packages/demisto_sdk/commands/common/hook_validations/pack_unique_files.py", line 878, in _is_right_version
    rn_version = self._get_pack_latest_rn_version()
  File "/Users/galnakash/.local/share/virtualenvs/content-NObhtT8p/lib/python3.10/site-packages/demisto_sdk/commands/common/hook_validations/pack_unique_files.py", line 210, in _get_pack_latest_rn_version
    list_of_versions = [
  File "/Users/galnakash/.local/share/virtualenvs/content-NObhtT8p/lib/python3.10/site-packages/demisto_sdk/commands/common/hook_validations/pack_unique_files.py", line 211, in <listcomp>
    rn[: rn.rindex(".")].replace("_", ".") for rn in list_of_release_notes
ValueError: substring not found
`

File: Packs/Reco/Integrations/Reco/Reco.py [hunk 2]
```diff
@@ -317,9 +505,29 @@ def main() -> None:
             )
             demisto.setLastRun(next_run)
             demisto.incidents(incidents)
+        elif command == "reco-update-incident-timeline":
+            reco_client.update_reco_incident_timeline(
+                incident_id=demisto.args()["incident_id"],
+                comment=demisto.args()["comment"],
+            )
+            return_results("Incident timeline updated successfully")
```
Comment (yucohen) [inline Packs/Reco/Integrations/Reco/Reco.py:513]: the return_results should get a `CommandResults` object
So maybe change to something like :
return_results(CommandResults(readable_output="Incident timeline updated successfully"))
Also worth adding the raw response as discussed in the demo

Comment (GalNakash-RecoLabs) [reply Packs/Reco/Integrations/Reco/Reco.py]: Done

File: Packs/Reco/Integrations/Reco/Reco.py [hunk 3]
```diff
@@ -317,9 +506,32 @@ def main() -> None:
...
+            return_results(CommandResults(
+                raw_response=response,
+                readable_output=f"Timeline updated successfully for incident {incident_id}"))
+        elif command == "reco-resolve-visibility-event":
+            entity_id = demisto.args()["entity_id"]
+            label_name = demisto.args()["label_name"]
+            reco_client.resolve_visibility_event(
+                entity_id=entity_id,
+                label_name=label_name)
+            return_results(f"Visibility event {entity_id} resolved successfully")
```
Comment (yucohen) [inline Packs/Reco/Integrations/Reco/Reco.py:524]: same here with the commandResults

Comment (GalNakash-RecoLabs) [reply Packs/Reco/Integrations/Reco/Reco.py]: Done

**Comment by yucohen** [comment]
@GalNakash-RecoLabs @edik24 I'm going to merge to internal PR and make some minor changes, as I dont have permissions to push to this fork and we want it to be merged before the weekend

**Pull request merged**